rust/src/transformations/impute/mod.rs

```diff
@@ -3,34 +3,46 @@ mod ffi;
 
 use opendp_derive::bootstrap;
 
-use crate::core::{Domain, Function, StabilityMap, Transformation};
+use crate::core::{Domain, Function, StabilityMap, Transformation, MetricSpace};
 use crate::domains::{AtomDomain, OptionDomain, VectorDomain};
 use crate::error::Fallible;
-use crate::metrics::SymmetricDistance;
 use crate::traits::samplers::SampleUniform;
 use crate::traits::{CheckAtom, CheckNull, Float, InherentNull};
 use crate::transformations::{make_row_by_row, make_row_by_row_fallible};
 
-#[bootstrap(features("contrib"), generics(TA(example = "$get_first(bounds)")))]
+use super::DatasetMetric;
+
+#[bootstrap(
+    features("contrib"), 
+    generics(M(suppress), TA(suppress)),
+    derived_types(TA = "$get_atom(get_type(input_domain))")
+)]
 /// Make a Transformation that replaces NaN values in `Vec<TA>` with uniformly distributed floats within `bounds`.
 ///
 /// # Arguments
+/// * `input_domain` - Domain of the input.
```

Comment:
Do you want a table like other constructors above here?

Reply:
There's only one valid input domain in this case, so the table feels kind of silly.